code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -415,7 +415,7 @@
 
 	if(istype(AM, /obj/structure/bed)) //This is only necessary because of rollerbeds and swivel chairs.
 		var/obj/structure/bed/B = AM
-		if(B.locked_atoms.len)
+		if(ISREALLIST(B))
```

Comment:
@ComicIronic That'd work but that doesn't check its length or if it's a list. That's why this macro exists